.etc/golangci.yml

```diff
@@ -39,13 +39,28 @@ linters:
     - whitespace
 
 linters-settings:
+  depguard:
+    rules:
+      main:
+        files:
+          - $all
+        allow:
+          - $gostd
+          - github.com/bwesterb/go-ristretto
+          - github.com/cloudflare/circl
   funlen:
     lines: 120
     statements: 80
   nestif:
     min-complexity: 6
+  goconst:
+    ignore-tests: true
```

Comment:
Addresses
```
::error file=group/group_test.go,line=27,col=11::string `/Mul` has 3 occurrences, make it a constant (goconst)
::error file=group/group_test.go,line=25,col=11::string `/Add` has 3 occurrences, make it a constant (goconst)
```

Reply:
we can add constants as is suggested, instead of ignoring everything.